templates/core/block/block--block-news-events-related-to-topic.html.twig

```diff
@@ -0,0 +1,44 @@
+{#
```

Comment:
Does this belong in gent_base? Or is it also to project specific?

Reply:
This is not the 'related articles' component we have in styleguide?

Reply:
yes, but in this case: it's specifically related to the `topic` content type, so it should be moved into the project 🙂